cartridge.lua

```diff
@@ -804,6 +805,27 @@ local function cfg(opts, box_opts)
     if type(box.cfg) == 'function' then
         confapplier.log_bootinfo()
     end
+    local crg_opts_to_logs = table.deepcopy(opts)
```

Comment:
The code looks correct as for me but there are several questions:
- Do we really need to show box.cfg (not box.info) that contains only user-defined opinions and doesn't show defaults.
- Is json format usable (especially there are a lot of options and user uses log_format json)
- There is still consern about security. We hide replication passwords but there is no guarantee that such options won't be introduced in future. E.g. in scope of multilisten + encryption features.